index.html

```diff
@@ -79,23 +79,77 @@ <h1>Avatar</h1>
                 <img src="./img/hacker.png"alt="avatar-extraLarge" />
               </div>
             </div>
+            
+
+
+
+
             <div class="code-container">
               <iframe
                 src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2509%253Cdiv%2520class%253D%2522avatar-sm%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522...%2522%2520alt%253D%2522avatar-small%2522%2520%252F%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522avatar-md%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522...%2522%2520alt%253D%2522avatar-medium%2522%2520%252F%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522avatar-l%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522...%2522%2520alt%253D%2522avatar-large%2522%2520%252F%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522avatar-xl%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522...%2522%2520alt%253D%2522avatar-extraLarge%2522%2520%252F%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E"
-                style="
-                  width: 555px;
-                  height: 409px;
-                  border: 0;
-                  transform: scale(1);
-                  overflow: hidden;
-                "
+                style="width: 766px; height: 502px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
This style is duplicated in next iframe as well. What can be the solution to avoid this duplication?